custom_dc/index.md

```diff
@@ -37,11 +37,10 @@ A custom Data Commons site is deployed in Google Cloud Platform (GCP). The owner
 
 If you have the resources to develop and maintain a custom Data Commons instance, this is a good option for the following use cases:
 
+- You want to host your data on your own website, and take advantage of Data Commons natural-language query interface, and exploration and visualization tools. (You may host all or part of your data on the main public Data Commons site as well.)
```

Comment:
Can you break this into bullet points like before?

Reply:
Sorry - it is bulleted.

Just remove the main DC comment and we're good.

Reply:
Done.